build.sbt

```diff
@@ -73,8 +73,9 @@ scalacOptions ++= List(
 enablePlugins(ScriptedPlugin)
 sbtPlugin := true
 scriptedBufferLog := false
+scriptedBatchExecution := true
+scriptedParallelInstances := 2
```

Comment:
I tried 4 initially, but it brings transient failures on CI (every other run roughly...)